languages/Spanish

```diff
@@ -0,0 +1,134 @@
+1=Zona de batalla
+2=¡Sí! Ganaste el juego. ¡Pronto se agregarán nuevos niveles! ¡Oh, olvidé decirte algo! Tu puntaje fue
+3=.
+4=Comienza el juego
+5=Jugar en modo Realidad
+6=Cargar el último juego guardado
+7=Leer instrucciones
+8=Cambiar opciones
+9=Salir
+10=Menú principal de Battle Zone. Seleccione un elemento del menú.
+11=Original: "Dark Mark-a-Tark"
+12=Canción de mensaje de error de Windows XP: "MetaNights Revenge"
+13=Tech House
+14=Techno Chase
+15=Lazo orquestal 1
+16=Caza de tensión
+17=Música aleatoria que cambia entre niveles
+18=Apaga la música
+19=¡Seleccione la música de fondo que desea usar!
+20=Cambiar música de fondo
+21=Modo de teclado
+22=Seleccionar género de enemigos
+23=Seleccionar sonido de espada
+24=Toggle Enemy Attack Taunt Sounds
```

Comment:
Needs to be translated?

Reply:
Woops? I should have checked the Spanish file more thoroughly. Then again, it looks like this pull request is a few commits behind, so I may need to send you another. Thanks for pointing that out.